src/solvers/td3.ml

```diff
@@ -131,6 +131,8 @@ module WP =
       (* If true, incremental side-effected var restart will only restart destabilized globals (using hack).
          If false, it will restart all destabilized side-effected vars. *)
       let restart_only_globals = GobConfig.get_bool "incremental.restart.sided.only-global" in
+      let restart_only_access = GobConfig.get_bool "incremental.restart.sided.only-access" in
```

Comment:
With more of these options, the nicer thing would be to either have one option with three string values if they can be ordered such that they subsume each other, or have to non-only options (so negated) if all combinations are reasonable.
But we can leave it here as-is, the options need to be cleaned up before #391 gets merged to `master` anyway.